doc/topics/transports/tcp.rst

```diff
@@ -13,6 +13,12 @@ to ``tcp`` on each Salt minion and Salt master.
 
    transport: tcp
 
+.. warning::
+
+    We currently recommend that when using Syndics that all masters and minions
```

Comment:
I suppose `masters` and `minions` should be capitalized here as well, if `Syndics` is.